packages/studio-web/src/app/upload.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from "@angular/core";
+import { BehaviorSubject } from "rxjs";
+
+@Injectable({
+  providedIn: "root",
+})
+export class UploadService {
+  $currentAudio = new BehaviorSubject<Blob | File | null>(null);
+  $currentText = new BehaviorSubject<Blob | string | null>(null);
```

Comment:
What does the convention say to do with the `$` anyway?  Before or after the name?  Does it depend if it's a `Subject` or an `Observable`?

Reply:
I thought a $ prefix was subject and a $ suffix was observable. But it looks like Angular only explicitly documents the use of $ as a suffix for observables.

Reply:
Hmm - now that I think of it, `Subject`s are also `Observable`s, so that makes sense.

Reply:
true, but I kind of like having $ prefix since it lets you know that you can call `next` on it etc.. but I don't feel super strongly about it. I just changed it to $textInput but I can change it if you prefer

Reply:
Makes sense!  I agree it's useful to have a $ on it so we know what it is...